.github/workflows/lint.yml

```diff
@@ -23,3 +24,14 @@ jobs:
 
       - name: Lint
         uses: golangci/golangci-lint-action@v3
+        with:
+          # Specify a version instead of 'latest'
+          # because of something wrong with 1.48.0
+          version: v1.50.1
+          args:
+            --config=./.golangci.yml
```

Comment:
Is this already implied by the binary?

Reply:
makefile command if only for local run. using golint-ci action is faster as far as doesn't require downloading and installing golint-ci